sdk/identity/azure-identity/src/main/java/com/azure/identity/implementation/IdentityClient.java

```diff
@@ -206,6 +206,8 @@
             }
             String processOutput = output.toString();
 
+            process.waitFor();
```

Comment:
Should we have a timeout?

Reply:
We can probably make an educated guess of the longest possible time the sub process would take and fail after that?

Reply:
agreed this must have some default timeout. In .NET I set this to 10 seconds.

Reply:
added 10 seconds timeout